### PR TITLE
Bump version to v1.3.2

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Bump version to v1.3.2.

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
@haiyuazhang

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,28 @@
 # Change Log
 
+## v1.3.2 (2021-01-26)
+
+### NEW MODULES
+  - azure_rm_route_info: Add azure_rm_route_info module ([#334](https://github.com/ansible-collections/azure/pull/334))
```
Comment (haiyuazhang) [inline CHANGELOG.md]:
```suggestion
  - azure_rm_route_info: add azure_rm_route_info module ([#334](https://github.com/ansible-collections/azure/pull/334))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+
+### FEATURE ENHANCEMENT
+  - azure_rm_postgresqlserver: added storage_autogrow option to postgresqlserver ([#387](https://github.com/ansible-collections/azure/pull/387))
```
Comment (haiyuazhang) [inline CHANGELOG.md]:
```suggestion
  - azure_rm_postgresqlserver: add storage_autogrow option to postgresqlserver ([#387](https://github.com/ansible-collections/azure/pull/387))
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,28 @@
...
+  - azure_rm_keyvaultsecret: Added content type parameter to azure_rm_keyvaultsecret ([#317](https://github.com/ansible-collections/azure/pull/317))
```
Comment (haiyuazhang) [inline CHANGELOG.md]:
```suggestion
  - azure_rm_keyvaultsecret: add content type parameter to azure_rm_keyvaultsecret ([#317](https://github.com/ansible-collections/azure/pull/317))
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,28 @@
...
+  - azure_rm_keyvaultsecret_info: Added content type parameter to azure_rm_keyvaultsecret ([#317](https://github.com/ansible-collections/azure/pull/317))
```
Comment (haiyuazhang) [inline CHANGELOG.md]:
```suggestion
  - azure_rm_keyvaultsecret_info: add content type parameter to azure_rm_keyvaultsecret ([#317](https://github.com/ansible-collections/azure/pull/317))
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,28 @@
...
+  - azure_rm_mysqlserver: Add missing Mysql version 8.0 ([#319](https://github.com/ansible-collections/azure/pull/319))
```
Comment (haiyuazhang) [inline CHANGELOG.md]:
```suggestion
  - azure_rm_mysqlserver: add missing Mysql version 8.0 ([#319](https://github.com/ansible-collections/azure/pull/319))
```

File: CHANGELOG.md [hunk 6]
```diff
@@ -1,5 +1,28 @@
...
+
+### BUG FIXING
+  - Test_unit: add resource group for datalake store testing in ado pipeline ([#375](https://github.com/ansible-collections/azure/pull/375))
+  - README.md: Update README to include a link to documentation ([#376](https://github.com/ansible-collections/azure/pull/376))
```
Comment (haiyuazhang) [inline CHANGELOG.md]:
```suggestion
  - README.md: update README to include a link to documentation ([#376](https://github.com/ansible-collections/azure/pull/376))
```

File: CHANGELOG.md [hunk 7]
```diff
@@ -1,5 +1,28 @@
...
+  - azure_rm_deployment: Update azure_rm_deployment document ([#384](https://github.com/ansible-collections/azure/pull/384))
```
Comment (haiyuazhang) [inline CHANGELOG.md]:
```suggestion
  - azure_rm_deployment: update azure_rm_deployment document ([#384](https://github.com/ansible-collections/azure/pull/384))
```

File: CHANGELOG.md [hunk 8]
```diff
@@ -1,5 +1,28 @@
...
+  - azure_rm_azurefirewall: added support for tags in exec_module ([#360](https://github.com/ansible-collections/azure/pull/360))
```
Comment (haiyuazhang) [inline CHANGELOG.md]:
```suggestion
  - azure_rm_azurefirewall: add support for tags in exec_module ([#360](https://github.com/ansible-collections/azure/pull/360))
```

**Pull request merged**